app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/HomeController.scala

```diff
@@ -23,14 +23,12 @@ import uk.gov.hmrc.cgtpropertydisposalsfrontend.views
 import uk.gov.hmrc.play.bootstrap.controller.FrontendController
 
 @Singleton
-class ReturnsController @Inject()(
+class HomeController @Inject()(
   cc: MessagesControllerComponents,
   startPage: views.html.returns.start
 )(implicit viewConfig: ViewConfig)
     extends FrontendController(cc) {
-
   def start(): Action[AnyContent] = Action { implicit request =>
```

Comment:
we should only show this page if we're in the `JourneyStatus.Subscribed` state - we should go to the start endpoint otherwise. There's a method called `redirectToStartWhenInvalidJourney` in the unit tests to facilitate testing this.

Reply:
```scala
  sealed trait SubscriptionComplete extends JourneyStatus with Product with Serializable

  object SubscriptionComplete {
    // subscription has been submitted to ETMP
    final case class Subscribed(
      subscriptionDetails: SubscriptionDetails,
      subscriptionResponse: SubscriptionResponse
    ) extends SubscriptionStatus
  }
```

Reply:
OK, I will create a trait `WithSubscriptionCompleteAction` and mix that in to the controller.